idaes/commands/extensions.py

```diff
@@ -152,3 +181,41 @@ def get_extensions(
             print_extensions_version(library_only)
     else:
         click.echo("\n* You must provide a download URL for IDAES binary files.")
+
+
+@cb.command(name="hash-extensions", help="Calculate release hashes")
+@click.option(
+    "--release",
+    help="Optional, specify an official binary release to download",
+    default=None,
+    required=True)
+@click.option(
+    "--path",
+    help="Directory of release files",
+    default=None,
+    required=True)
+def hash_extensions(release, path):
+    hfile = f"sha256sum_{release}.txt"
+    if path is not None:
+        hfile = os.path.join(path, hfile)
+
+    def _w(fp, pack, plat):
+        f = f"idaes-{pack}-{plat}-64.tar.gz"
+        if path is not None:
+            h = idaes.util.download_bin.hash_file_sha256(os.path.join(path, f))
+        else:
+            h = idaes.util.download_bin.hash_file_sha256(f)
+        fp.write(h)
```

Comment:
Is this a format that will be read elsewhere? Maybe it would be easier to just write it as JSON? I may be misunderstanding the purpose of this function.

Reply:
You are right.  The problem is previous to this I used a bash script to do these, and I just used the simple space separated format.  I'd definitely use json if not for the old files.